src/users/tests/test_api.py

```diff
@@ -2,41 +2,61 @@
 from django.contrib.auth.models import Group
 from django.urls import reverse
 
+from core.models import Token
+
+
+@pytest.fixture
+def auth_user(django_user_model):
+    return django_user_model.objects.create(
+        email="test_auth@example.com",
+        speaker_name="Auth User",
+        verified=True,
+        is_active=True,
+    )
+
+
+@pytest.fixture
+def api_client_with_auth_user(api_client, auth_user):
+    token, _ = Token.objects.get_or_create(user=auth_user)
+    api_client.credentials(HTTP_AUTHORIZATION=f"Token {token.key}")
+    return api_client
+
 
 @pytest.mark.django_db
-def test_user_list_with_role_filter_exact_match(api_client, django_user_model):
-    group = Group.objects.create(name='Reviewer')
-    user = django_user_model.objects.create(
+def test_user_list_with_role_filter_exact_match(api_client_with_auth_user, django_user_model):
```

Comment:
不能直接用既有 fixture `api_client` ([ref](https://github.com/pycontw/pycon.tw/blob/fb65ccb897217e404037667dbefca3c767c5c3fa/src/conftest.py#L209-L214)) 嗎？api client fixture 應該不用自己再實作一遍，只是這個 fixture 也許需要更名一下會比較清楚

```suggestion
def test_user_list_with_role_filter_exact_match(api_client, django_user_model):
```